starter-dev-backend/src/schema/github/github.typedefs.ts

```diff
@@ -21,14 +21,14 @@ export const githubTypeDefs = gql`
   """
   A Owner object used in Repo
   """
```

Comment:
Suggestion: Update comment to reflect schema change.

```suggestion
  """
  A User object used in Repo
  """
```